ska_conda/pkg_defs/astropy/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "3.0.3" %}
+
+package:
+  name: astropy
+  version: {{version}}
+
+source:
+  fn: astropy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/a/astropy/astropy-{{version}}.tar.gz
+  sha256: 6af07abe5e263820a3dec93832a6ad74005013071490e125afbc6514411721da
+
+
+build:
+  detect_binary_files_with_prefix: False
+  entry_points:
+    - fits2bitmap = astropy.visualization.scripts.fits2bitmap:main
+    - fitscheck = astropy.io.fits.scripts.fitscheck:main
+    - fitsdiff = astropy.io.fits.scripts.fitsdiff:main
+    - fitsheader = astropy.io.fits.scripts.fitsheader:main
+    - fitsinfo = astropy.io.fits.scripts.fitsinfo:main
+    - samp_hub = astropy.samp.hub_script:hub_script
+    - volint = astropy.io.votable.volint:main
+    - wcslint = astropy.wcs.wcslint:main
+  number: 1
+
+requirements:
+
+  build:
+    - python
+    - setuptools
+    - numpy ==1.12.1
+    - llvmlite ==0.18.0
```

Comment:
I changed these to be very specific to our install.  I think this works out so that our recipe/astropy build will be naturally deprecated when we no longer have these pins.